spyder/plugins/editor/widgets/editor.py

```diff
@@ -624,6 +635,27 @@ def show_in_external_file_explorer(self, fnames=None):
             fnames = self.get_current_filename()
         show_in_external_file_explorer(fnames)
 
+    def copy_absolute_path(self):
+        QApplication.clipboard().setText(
+            self.get_current_filename())
+
+    def copy_relative_path(self):
+        file_path = os.path.splitdrive(self.get_current_filename())[0]

```

Comment:
Let's change this to

```python
file_drive = osp.splitdrive(self.get_current_filename())[0]
```

* We're getting the drive of the current file here, not its path.
* `osp` is the alias we use for `os.path` almost everywhere in Spyder.